### ISSUE TITLE
Should at-rule-semicolon-newline-after be boolean?

### ISSUE THREAD

**Issue by samjbmason** [stylelint/stylelint#2108]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?
A bug where the rule `at-rule-semicolon-newline-after` should have the value of `true` rather than `"always"`

> Which rule, if any, is this issue related to?
`at-rule-semicolon-newline-after`

> What CSS is needed to reproduce this issue?
```css
@import url("x.css");
@import url("y.css");
```

> What stylelint configuration is needed to reproduce this issue?
```json
{
  "rules": {
    "at-rule-semicolon-newline-after": "always"
  }
}
```

> Which version of stylelint are you using?
`7.6.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?
CLI

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?
No

> What did you expect to happen?
As there is only one option it feels like it should be boolean

> What actually happened (e.g. what warnings or errors you are getting)?
I passed it the "always" string

**Comment by evilebottnawi** [comment]
@samjbmason we don't request from world to add `never` options, but theoretically it is realy

**Comment by davidtheclark** [comment]
Nope, we have very standardized options, and `always` matches the convention across many rules. `true` is only used for `*-no-*` rules.

**Comment by evilebottnawi** [comment]
@davidtheclark maybe add this note in development docs?

**Comment by evilebottnawi** [comment]
oh, sorry, we alread have this
> Use explicit, rather than implicit, options. For example, color-hex-case: "upper"|"lower" rather than color-hex-uppercase: "always"|"never". As color-hex-uppercase: "never" implies always lowercase, whereas color-hex-case: "lower" makes it explicit.

**Comment by evilebottnawi** [comment]
But not have note about standardized :sob:
